css/headerFooter.scss

```diff
@@ -0,0 +1,135 @@
+@mixin sairaFont($size) {
+  font-family: "Saira", sans-serif;
+  font-optical-sizing: auto;
+  font-weight: 700;
+  font-size: $size;
+  font-style: normal;
+  font-variation-settings: "wdth" 100;
+}
+@mixin topLink {
+  text-decoration: none;
+  color: white;
+}
+
+.calibriFont {
+  font-family: Calibri, Candara, Segoe, "Segoe UI", Optima, Arial, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 20px;
+}
+
+.header {
```

Comment:
Currently the content does not occupy the whole page. Check how can we achieve that and to set the footer to the bottom. Maybe position:fixed will be a better solution :)

![Capture](https://github.com/user-attachments/assets/8821ad65-178d-48ab-aec0-60347ffe0996)

Reply:
I have a problem here. If I make the position fixed, main content is not stretched up to the bottom of the page. How to deal with it?

Reply:
<img width="887" alt="image" src="https://github.com/user-attachments/assets/a727677e-f729-4262-a802-b20ac2bf6ca8">

Reply:
hm...you know, I think I've got it with this property
min-height: 100vh;